style.css

```diff
@@ -0,0 +1,308 @@
+* {
+    padding:0px;
+    margin: 0px;
+    font-family: "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", 
+  Helvetica, Arial, "Lucida Grande", sans-serif;
+}
+
+.main-grid {
+    display:grid;
+    height: 100vh;
+    grid-template-columns: 180px 180px 720px 360px; 
+}
+
+@media screen and (max-width:987px) {
+    .main-grid {
+        grid-template-columns: 180px 0px 720px 360px;                      
+    }
+    .left-menu {
+        display:none;
+    }
+}
+
+/* --------------------------------LEFT COLUMN 1--------------------------------*/
+
+.left-side-icons {
+    background-color: rgb(255, 255, 255);
+}
+
+.left-icons-wrapper {
+    display:grid;
+    row-gap: 5px;
+    align-items: center;
+    justify-items: right;
+    padding-top: 5px;
+        
+}
+
+img {
+    padding: 10px;
+    min-height: 20px;
+    /*border: solid 2px blueviolet;
+    border-radius:5px*/
+}
+
+#skull, #idea, #rocket, #connections, #spider, #sand-castle, #jellyfish, #crab, #wasp {
+    width:40px;
+     
```

Comment:
unnecessary space.. looks like this happens throughout

Reply:
The CSS spaces were definitely where I had been experimenting and taking things out and then not gone over to remove the space. I've removed that space before my final submission.